### PR TITLE
Newsletters: add 205 (2022-06-22)

### PR DESCRIPTION
**Pull request by harding**
- [x] Clients/services section @bitschmidty 
- [x] Releases/RCs and topics @harding

### CODE REVIEW

File: _posts/en/newsletters/2022-06-22-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+- [HWI #611][] adds single-sig support for [bech32m addresses][topic
+  bech32] with the BitBox02 hardware signing device.
+
+{% include references.md %}
+{% include linkers/issues.md v=2 issues="602,2224,611,25353,25373" %}
+[lnd 0.15.0-beta.rc6]: https://github.com/lightningnetwork/lnd/releases/tag/v0.15.0-beta.rc6
+[rbf discussion]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-June/020557.html
+[hertzbleed]: https://www.hertzbleed.com/
+[news116 sponsorship]: /en/newsletters/2020/09/23/#transaction-fee-sponsorship
+[poelstra timestamping]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-June/020569.htmla
```
Comment (johnzweng) [inline _posts/en/newsletters/2022-06-22-newsletter.md:233]: Small typo in file suffix: *.html**a**
```suggestion
[poelstra timestamping]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2022-June/020569.html
```

**Review by johnzweng — CHANGES_REQUESTED** [review]

File: _posts/en/newsletters/2022-06-22-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+slug: 2022-06-22-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposed option for Bitcoin Core that
+would make it easier to enable transaction replacement even for
+transactions that don't opt-in to BIP125, links to information about the
+Hertzbleed sidechannel vulnerability, summarizes the conclusion of a
+discussion about time stamping system design, and examines an new
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-06-22-newsletter.md:14]:
```suggestion
discussion about time stamping system design, and examines a new
```

File: _posts/en/newsletters/2022-06-22-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+
+  Exploiting the vulnerability would require an attacker to measure
+  either the power consumption of a CPU chip or measure the duration
+  of parts of the signing operation.  Ideally for an attacker, they
+  would be able to take measurements while a user creates many
+  signatures using the same private key.  As such, the vulnerability is
+  more likely to affect frequently used hot wallets, such as those used
+  by hosted services and LN routing nodes.  Mostly or entirely offline
+  wallets that are used in secure environments would be much more
+  resistant to attacks.
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-06-22-newsletter.md:68]: Would I understand correctly that address reuse exacerbates this? Perhaps we could call that out specifically to add to the well-known reasons for why address reuse is a bad practice.

File: _posts/en/newsletters/2022-06-22-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+
+- **New RIDDLE anti-sybil method:** Adam "Waxwing" Gibson
+  [posted][gibson riddle post] to the Bitcoin-Dev mailing list a
+  [proposal][gibson riddle gist] for an [anti-sybil][sybil] mechanism
+  that uses the Bitcoin UTXO set and which can provide reasonably good
+  privacy.  A user can generate a list of UTXOs where one of the UTXOs
+  belongs to the user and the rest belong to other users.  The user then
+  creates a signature which third parties can verify came from an owner
+  of one of the UTXOs---but which the verifier can't determine which of
+  the listed UTXO owners created it.
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-06-22-newsletter.md:130]: Too many witches spoil the potion—that last sentence is a bit hard to parse. Perhaps:

```suggestion
  belongs to the user and the rest belong to other users.  The user then
  creates a signature which provably came from an owner
  of one of the listed UTXOs, but does not reveal which owner created it.
  ```

File: _posts/en/newsletters/2022-06-22-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+  Additionally, high value UTXOs could be treated as multiple UTXOs of a
+  lower value, so a 10 BTC UTXO could allow a user to sign up for 10
+  different accounts at different services each requiring 1 BTC of
+  capital in the global context.
+
+  Although the RIDDLE protocol does provide privacy advantages over
+  other anti-sybil mechanisms, Gibson does warn that information from
+  use of the system can be combined with other available information to
+  potentially reduce the users privacy.  He writes, "there is no
```
Comment (Xekyo) [inline _posts/en/newsletters/2022-06-22-newsletter.md:162]:
```suggestion
  potentially reduce the user's privacy.  He writes, "there is no
```

**Review by Xekyo — COMMENTED** [review]
Thanks that was a great read

**Comment by harding** [comment]
Made edits (thanks @johnzweng and @Xekyo ), added releases, added topic links (just one this week), and reviewed @bitschmidty 's contribution (thanks!).  Thanks everyone!

**Comment by bitschmidty** [comment]
Merged! Thanks @harding @Xekyo @johnzweng for contributing!

**Pull request merged**